cognite_toolkit/_cdf_tk/commands/_migrate/data_classes.py

```diff
@@ -12,9 +12,10 @@
     ToolkitValueError,
 )
 from cognite_toolkit._cdf_tk.storageio._data_classes import ModelList
+from cognite_toolkit._cdf_tk.utils.useful_types import AssetCentric
 
 
-class MigrationMapping(BaseModel, alias_generator=to_camel_case, extra="ignore"):
+class MigrationMapping(BaseModel, alias_generator=to_camel_case, extra="ignore", populate_by_name=True):
```

Comment:
Earlier MigrationMapping were only created form a `csv` file, which we only use the alias. However, now we need to create migratoin mapping in python code and thus would like to use snake case and not camel.